### ISSUE TITLE
Minimum contrast ratio between the text and background?

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/earthworks#1432]
This is being triggered as a false positive on a few elements, including the top navigation (Bookmarks, Search history, etc.), tools (Cite, Email, etcl) and the bottom right links in the Leaflet map. If you rescan the page, most of them seem to go away (known SI issue), except the links in the the Leaflet map. The white background behind the links uses transparency (it's set to 80%), which might be causing it, but it seems to pass against the gray map background. See the screenshot from this site - https://webaim.org/resources/linkcontrastchecker/.   

<img width="600" alt="Screenshot 2024-10-08 at 5 24 19 PM" src="https://github.com/user-attachments/assets/65496cd3-4651-49ea-a3a8-92d337fb5ba0">

However, depending on how a user zooms and/or pans the map, there might be circumstances where the background, such as the green block in this example -  https://earthworks.stanford.edu/catalog/stanford-qr604qw5282 - where it can be a valid issue. See screenshot.

<img width="390" alt="Screenshot 2024-10-08 at 5 45 57 PM" src="https://github.com/user-attachments/assets/c9965183-4166-40bb-ac04-9f60f152b347">

I recommend a solid white background with no transparency.

Once you [login to SiteImprove](https://siteimprove.stanford.edu), you can see the issue here - https://my2.siteimprove.com/Accessibility/1064059/NextGen/Issue/1?conformance=&siteTargetIssueKinds=1,2&wcagVersion=22&pageSegments=&ruleName=sia-r69&ruleId=69&issueKind=2&exceptTags=1,2&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2.

**Comment by thatbudakguy** [comment]
Leaflet's own styles add the transparency; not sure why. To restore a full white background, I think the CSS is:
```css
.leaflet-container .leaflet-control-attribution {
  background: white;
}
```

**Comment by thatbudakguy** [comment]
Added a fix to Geoblacklight. Once merged, we will need to cut a patch release and pull it into Earthworks (and we should probably push a new version of the styles package to NPM as well).

**Comment by corylown** [comment]
this is done in geoblacklight and needs to be pulled in.

**Comment by thatbudakguy** [comment]
to be more specific:
- tests on https://github.com/sul-dlss/earthworks/actions/runs/11333985017/job/31519231910?pr=1435 need to be fixed
- pr needs to be merged
- asset package needs to be published to npm
- then update earthworks to gbl v5.0.0alpha.11
